code/modules/mob/living/simple_animal/hostile/reenactors.dm

```diff
@@ -57,24 +57,30 @@
 	melee_damage_upper = 30
 	environment_smash = 2
 	attacktext = "slams"
-	deathmessage = "The Reenactor's body collapses in on itself from the strain!"
+	deathmessage = "The Gestapo's body collapses in on itself from the strain!"
 	loot = list(/obj/effect/gibspawner/human)
 
 /mob/living/simple_animal/hostile/wwii/melee/AttackingTarget()
 	..()
 	if(iscarbon(target))
-		var/mob/living/carbon/C = target
-		if(prob(40))
-			C.Weaken(3)
-			C.adjustBruteLoss(10)
-			C.visible_message("<span class='danger'>\The [src] smashes \the [C] into the ground!</span>", \
-					"<span class='userdanger'>\The [src] smashes you into the ground!</span>")
-			src.say(pick("RAAAAAGGHHHH!!!","AAAARRRGGHHHH!!!","RRRAAUUUGGHH!!!"))
+		var/mob/living/C = target
+		if(C.stat == DEAD)
+			C.gib()
+			visible_message("<span class='danger'>[src] devours [C] whole!</span>")
+			src << "<span class='userdanger'>You devour [C] whole, restoring your health!</span>"
```

Comment:
no that uses an L for the targeted mob I use a C its completely different sheesh